public/manifest.json

```diff
@@ -1,6 +1,7 @@
 {
-  "short_name": "Uniswap",
-  "name": "Uniswap",
+  "background_color": "#fff",
+  "display": "standalone",
+  "homepage_url": "https://app.uniswap.org",
```

Comment:
hmm can you remind me what this effects?

because sometimes the host isn't app.uniswap.org, e.g.
https://app-uniswap-org.ipns.dweb.link

Reply:
https://developer.chrome.com/docs/extensions/mv2/manifest/homepage_url/

It looks like it's really only useful for installable browser extensions. 🤷

Reply:
Maybe just drop it, no need for the extra reference to a particular URL